src/Tickets/Seating/app/blockEditor/hook-callbacks.js

```diff
@@ -153,12 +154,22 @@ export const filterSeatedTicketsAvailabilityMappedProps = (mappedProps) => {
  */
 export const filterSettingsFields = (fields) => {
 	const store = select(storeName);
-	const currentLayout = store.getCurrentLayoutId();
-	const layouts = store.getLayoutsInOptionFormat();
 
+	if ( store.isServiceStatusOk() ) {
```

Comment:
This check covers both a service outage, an invalid/expired license and an actual not inserted license.
Should **all** these cases trigger the appearance of the upsell? Or should the service outage be handled differently?